client/ios/Podfile.lock

```diff
@@ -357,6 +357,6 @@ SPEC CHECKSUMS:
   RNScreens: f28b48b8345f2f5f39ed6195518291515032a788
```

Comment:
why is this file changed? are you sure this should be in the commit?

Reply:
Yes.